src/data_generators/tests/copula_data_set_test.py

```diff
@@ -58,6 +61,11 @@ def test_to_impressions(self):
         np.testing.assert_equal(res[0], expected[0])
         np.testing.assert_equal(res[1], expected[1])
```

Comment:
Can lines 61--62 be combined into one line, i.e., np.testing.assert_equal(res, expected)?